wcvendors/wcvendors
# Updated: Setup wizard updates

### All Submissions:

### Changes proposed in this Pull Request:

Cleaning up our extensions pages and new go pro page. 

Closes #713.

### How to test the changes in this Pull Request:

1. Load WC Vendors Marketplace without Pro activated
2. See new pages and design changes. 
3.
